src/elements/hx-drawer/index.spec.js

```diff
@@ -0,0 +1,152 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-drawer> component tests
+ *
+ * @type HXDrawerElement
+ *
+ */
+describe('<hx-drawer> component tests', () => {
+    const template = '<hx-drawer>';
+
+    describe('test instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXDrawerElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXDrawerElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXDrawerElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+    });
+
+    describe('test Shadow DOM', () => {
+        describe('verify render', () => {
+            it('should have a static Shadow DOM', async function () {
+                const component = /** @type { HXDrawerElement } */ await fixture(template);
+                const shadow = component.shadowRoot.innerHTML;
+
+                expect(component).shadowDom.to.equal(shadow);
+            });
+
+            it('should render the Shadow Root mode open', async () => {
+                const component = /** @type { HXDrawerElement } */ await fixture(template);
+                const mode = component.shadowRoot.mode;
+
+                expect(mode).to.equal('open');
+            });
+
+            it('should have two <slot>s', async () => {
+                const component = /** @type { HXDrawerElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelectorAll('slot');
+                const len = query.length;
+
+                expect(len).to.be.equal(2);
+            });
+
+            it('should have an unnamed <slot>', async () => {
```

Comment:
Please add a test for the named slot `fixed`.

https://github.com/HelixDesignSystem/helix-ui/blob/09e990d69207c176926877731278f751615f401f/src/elements/hx-drawer/_shadow.html#L8

Reply:
Done